### PR TITLE
Fix issue-1170

### PR DESCRIPTION
**Pull request by dataclouder**
Fix Issue #1170, where an imported VM complains about missing or altered fields and Terraform tries to re-create the resource.

### CODE REVIEW

File: website/docs/r/vapp_vm.html.markdown [hunk 1]
```diff
@@ -570,6 +570,7 @@ example for usage details.
 * `inherited_metadata` - (*v3.11+*; *VCD 10.5.1+*) A map that contains read-only metadata that is automatically added by VCD (10.5.1+) and provides
   details on the origin of the VM (e.g. `vm.origin.id`, `vm.origin.name`, `vm.origin.type`).
 * `extra_config` - (*v3.13.+*) The VM extra configuration. See [Extra Configuration](#extra-configuration) for more detail.
+* `imported` - (*v3.13.+*) A true/false value telling whether the resource was imported.
```
Comment (lvirbalas) [inline website/docs/r/vapp_vm.html.markdown:573]: Would be great to have a description about in which cases the user should use this.

Comment (dataclouder) [reply website/docs/r/vapp_vm.html.markdown:573]: I think it's self-explanatory. If you have something in mind, please explain.

Comment (lvirbalas) [reply website/docs/r/vapp_vm.html.markdown:573]: I'm referring to the following logic:
https://github.com/vmware/terraform-provider-vcd/pull/1274/files#diff-e39bfe24504d3209d5395389ca0a05af237416e91c4f16a0e3fe56f299cf9b49R49-R51

We should expose that if user turns on `imported`, that it's not just an empty switch, but that the above logic gets triggered.

Comment (dataclouder) [reply website/docs/r/vapp_vm.html.markdown:573]: `imported` is a read-only attribute, not an argument. Users won't be able to change it.

Comment (lvirbalas) [reply website/docs/r/vapp_vm.html.markdown:573]: Oops! My bad, missed that it's computed. Thanks.

**Review by lvirbalas — APPROVED** [review]

**Review by adambarreiro — APPROVED** [review]

**Pull request merged**